pyproject.toml

```diff
@@ -102,6 +102,7 @@ docs = [
     "sphinx_design",
     "Jinja2>=3.1.3",
     "tomli; python_version < '3.11'",
+    "sphinxcontrib-globalsubs >= 0.1.1",
```

Comment:
One possible concern is that this is an early version of `sphinxcontrib-globalsubs`, which was released last year.  But, we've been using it in PlasmaPy for a while now with no problems, and it wouldn't be that much work to go back to `common_links.txt`.

Reply:
Since this only affects doc build and RTD is green, probably okay.